docs/1.0. Certificate Provisioning.md

```diff
@@ -147,32 +149,33 @@ The EST Server MUST present a valid TLS Server Certificate, this Certificate MUS
 
 ### EST Server - Client Authentication
 
-The EST Server MUST authenticate the EST Client that is requesting a TLS Certificate manually or automatically using a TLS Client Certificate.
+The EST Server MUST authenticate the EST Client that is requesting a TLS (Server?) Certificate manually or automatically using a TLS Client Certificate.
 
 The EST Server MUST support using a TLS Client Certificate, presented during the TLS handshake by the EST Client to authenticate if the EST Client is trusted. The TLS Client Certificate can either be signed by the current CA or a third party-trusted CA. The EST Server MUST check the validity of the EST Client's TLS Certificate before responding to its request, including checking the TLS certificate's revocation status where applicable.
 
-As the TLS Client Certificate is intended to be used for the lifetime of the device, the validity of the TLS Client Certificate and its corresponding chain of trust SHOULD exceed the intended lifetime of the product, it is therefore RECOMMENDED that this is a minimum of 30 years. The EST Server MAY ignore the validity period of the TLS Client Certificate and authenticate an EST Client using an expired TLS Client Certificate, if it has not been revoked.
+As the TLS Client Certificate is intended to be used for the lifetime of the device, the validity of the TLS Client Certificate and its corresponding chain of trust SHOULD exceed the intended lifetime of the product. It is therefore RECOMMENDED that this is a minimum of 30 years. The EST Server MAY ignore the validity period of the TLS Client Certificate and authenticate an EST Client using an expired TLS Client Certificate, if it has not been revoked.
 
-The EST Server MUST provide a method to load multiple trusted Root CA's, that are used to verify the TLS Client Certificate.
+The EST Server MUST provide a method to load multiple trusted Root CAs, that are used to verify the TLS Client Certificate.
 
 The EST Server MAY also support manual authentication of the EST Client if:
 - No TLS Client Certificate is presented during the TLS handshake
 - The TLS Client Certificate is not trusted
 - The local security policy requires an extra authentication step
 
 The exact process for manual authentication will be implementation specific, but the EST Server MUST provide enough information to the operator so they can authenticate the EST Client. During manual authentication the EST Server MUST respond with HTTP 202 and the response MUST include a `Retry-After` header as defined for HTTP 503 responses.
+(Worth linking to RFC 7030 Section 4.2.3 to be clear this is an invention of that spec, not NMOS? I had to check.)
```

Comment:
```suggestion
The exact process for manual authentication will be implementation specific, but the EST Server MUST provide enough information to the operator so they can authenticate the EST Client. In accordance with [Section 4.2.3 of RFC 7030](RFC-7030), during manual authentication the EST Server MUST respond with HTTP 202 and the response MUST include a `Retry-After` header as defined for HTTP 503 responses.
```

Reply:
@jonathanthorpe-sony @andrewbonney  What's the policy about linking to specific sections of RFCs? If we do a link directly here we can include the necessary fragment, but that pollutes the document a little?